hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -516,17 +505,52 @@ class MyStateMachine(RuleBasedStateMachine):
     drawn from this bundle will be consumed (as above) when requested.
     """
 
-    def __init__(self, name: str, *, consume: bool = False) -> None:
+    def __init__(self, name: str, *, consume: bool = False, transformations=()) -> None:
+        super().__init__(
+            (1, 2, 3), transformations=transformations
```

Comment:
```suggestion
            [...], transformations=transformations
```

(minor style nit) I think using the Ellipsis object here is a better signal that we're doing something special as a placehold.